custom/icds_reports/utils.py

```diff
@@ -373,7 +373,7 @@ def generate_data_for_map(data, loc_level, num_prop, denom_prop, fill_key_lower,
         on_map_name = row['%s_map_location_name' % loc_level] or name
         in_month = row[num_prop] or 0
 
-        value = row[num_prop] * 100 / (row[denom_prop] or 1)
+        value = (row[num_prop] or 0) * 100 / (row[denom_prop] or 1)
```

Comment:
Why not use the in_month variable defined just above this?

Can you write a test for this?

Reply:
Fixed, and added tests of calculate average in icds reports, in new PR:
https://github.com/dimagi/commcare-hq/pull/19262